ios/Sources/AboutFeature/ContributorCell.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+import Styleguide
```

Comment:
Sorry! This is an unnecessary fix!🙇‍♂️
But I don't know why I need sort.
What kind of problems can you think of?

Reply:
We don't care so much now, but I want to sort the import because it's not good if:


This is an example.
```swift
import SwiftUI
import Combine
import Styleguide
import AboutFeature
```

So I review about sorting.